python/ray/_private/node.py

```diff
@@ -889,16 +889,20 @@ def start_log_monitor(self):
             process_info,
         ]
 
-    def start_dashboard(self, require_dashboard: bool):
+    def start_api_server(self, *, include_dashboard: bool, raise_on_failure: bool):
         """Start the dashboard.
 
         Args:
-            require_dashboard: If true, this will raise an exception
+            include_dashboard: If true, this will load all modules of the
+                api server to start dashboard. Otherwise, it will only
+                start the modules that are not relevant to the dashboard.
+            raise_on_failure: If true, this will raise an exception
                 if we fail to start the dashboard. Otherwise it will print
```

Comment:
if we fail to start the api server or dashboard (since now they are different things)?